lib/android_build/maesdk/src/main/java/com/microsoft/applications/events/HttpClient.java

```diff
@@ -376,7 +394,8 @@ public FutureTask<Boolean> createTask(
       int[] header_index,
       byte[] header_buffer) {
     try {
-      Request r = new Request(this, url, method, body, request_id, header_index, header_buffer);
+      Map<String, String> headers = HttpClient.convertHeaders(header_index, header_buffer);
```

Comment:
Can we avoid copying request data from byte buffer to map, as this adds an performance overhead(though small) in the hot path of HTTP request processing ?